smart-engine/src/main/java/org/smartdata/server/engine/cmdlet/LocalCmdletExecutorService.java

```diff
@@ -42,11 +42,11 @@
 
 public class LocalCmdletExecutorService extends CmdletExecutorService implements StatusReporter {
   private static final Logger LOG = LoggerFactory.getLogger(LocalCmdletExecutorService.class);
-  private SmartConf conf;
+  private final SmartConf conf;
   private CmdletFactory cmdletFactory;
   private CmdletExecutor cmdletExecutor;
   private ScheduledExecutorService executorService;
-  private boolean disableLocalExec;
+  private final boolean disableLocalExec;
```

Comment:
pls consider sorting the class fields according to the following order:
1. private static final SomeType field;
2. private static SomeType field;
3. private final SomeType field;
4. private SomeType field;